__tests__/app.test.js

```diff
@@ -324,4 +324,35 @@ describe("app", () => {
         });
     });
   });
+
+  describe("GET: 200 - api/articles (sorting queries)", () => {
+    test("should sort articles by comment_count in ascending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=comment_count&order=ASC")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          expect(articles).toBeSortedBy("comment_count", { ascending: true });
+        });
+    });
+    test("GET: 200 - should sort articles by votes in descending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes&order=DESC")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          expect(articles).toBeSortedBy("votes", { descending: true });
+        });
+    });
+
+    test("GET: 200 - should return default sorting if invalid sort_by column is entered", () => {
+      return request(app)
+        .get("/api/articles?sort_by=invalid_col&order=DESC")
```

Comment:
Getting the articles sorted by default value created_at is OK, but this one should be a sad path 400 invalid sort_by query